packages/client/src/config.ts

```diff
@@ -1,6 +1,38 @@
 import { SetupContractConfig } from "@latticexyz/std-client";
 import { Wallet } from "ethers";
+import * as wagmiChains from "@wagmi/chains";
+
+// TODO: move this to a chains/config package
+const latticeTestnet = {
+  name: "Lattice Testnet",
+  id: 4242,
+  network: "lattice-testnet",
+  nativeCurrency: { decimals: 18, name: "Ether", symbol: "ETH" },
+  rpcUrls: {
+    default: {
+      http: ["https://follower.testnet-chain.linfra.xyz"],
+      webSocket: ["wss://follower.testnet-chain.linfra.xyz"],
+    },
+    public: {
+      http: ["https://follower.testnet-chain.linfra.xyz"],
+      webSocket: ["wss://follower.testnet-chain.linfra.xyz"],
+    },
+  },
+} satisfies wagmiChains.Chain;
+
+const chains = { ...wagmiChains, latticeTestnet };
+
+// Load chain config
+const env = import.meta.env;
```

Comment:
fwiw, you shouldn't alias `import.meta.env`, otherwise the values can't be replaced during vite's build process